tests/migration/conftest.py

```diff
@@ -21,60 +21,41 @@
     from sqlalchemy.engine import Connection, Engine
 
 
-pytest_plugins = [
-    "tests.fixtures.services",
-]
-
-
-@pytest.fixture(scope="function")
-def application(
-    services_fixture: ServicesFixture,
-) -> Generator[ApplicationFixture, None, None]:
-    app = ApplicationFixture.create()
-    yield app
-    app.close()
-
-
-@pytest.fixture(scope="function")
-def database(application: ApplicationFixture) -> Generator[DatabaseFixture, None, None]:
-    # This is very similar to the normal database fixture and uses the same object,
-    # but because these tests are done outside a transaction, we need this fixture
-    # to have function scope, so the database schema is completely reset between
-    # tests.
-    db = DatabaseFixture.create()
-    yield db
-    db.close()
+@pytest.fixture
+def alembic_config_path() -> Path:
+    return Path(__file__).parent.parent.parent.absolute() / "alembic.ini"
 
 
 @pytest.fixture
-def alembic_config() -> Config:
+def alembic_config(alembic_config_path: Path) -> Config:
     """
     Use an explicit path to the alembic config file. This lets us run pytest
     from a different directory than the root of the project.
     """
-    return Config(
-        config_options={
-            "file": str(Path(__file__).parent.parent.parent.absolute() / "alembic.ini")
-        }
-    )
+    return Config(config_options={"file": str(alembic_config_path)})
 
 
 @pytest.fixture
-def alembic_engine(database: DatabaseFixture) -> Engine:
+def alembic_engine(function_database: DatabaseFixture) -> Engine:
```

Comment:
Now that we have a fixture `function_database` that can provide a per test function database in the main test suite, we can use it here to run the alembic tests in an isolated environment as part of the main test suite.